libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3346,6 +3354,64 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 	});
 }
 
+string YulUtilFunctions::bytesToFixedBytesConversionFunction(ArrayType const& _from, FixedBytesType const& _to)
+{
+	solAssert(_from.isByteArray(), "");
+	solAssert(_from.isDynamicallySized(), "");
+	string functionName = "convert_bytes_to_bytesNN_from_" + _from.identifier() + "_to_" + _to.identifier();
```

Comment:
If we are using that in the function name (as well as the type internally is called that):
```suggestion
	string functionName = "convert_bytes_to_fixedbytes_from_" + _from.identifier() + "_to_" + _to.identifier();
```